pkg/utils/release/sbom.go

```diff
@@ -0,0 +1,86 @@
+/* SBOM of type structures matches Pyxis structure
```

Comment:
In the above example of StackOverFlow that it have one structure type, so this why it's made shorter , 
in our case I can reduce the SBOM types for now, but in the future if needed working with more fields then it will be added again.
now,  I cannot tell if we are going to use all fields but I thought it would be better to add all fields. 
WDYT?

Reply:
OK, I think it's fine to have it.